Rebus.LegacySample/packages.config

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Castle.Core" version="3.3.0" targetFramework="net45" />
+  <package id="Castle.Windsor" version="3.3.0" targetFramework="net45" />
+  <package id="CommonServiceLocator" version="1.3" targetFramework="net45" />
+  <package id="HermaFx.DataAnnotations" version="1.3.301" targetFramework="net45" />
+  <package id="HermaFx.Foundation" version="1.3.301" targetFramework="net45" />
+  <package id="HermaFx.Rebus" version="1.3.301" targetFramework="net45" />
```

Comment:
Do we need this ones for the sample projects? (not sure wether it can be removed)